schema/property/meta.json

```diff
@@ -0,0 +1,10 @@
+{
+    "schemaId": "property/meta",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Schema of material's meta properties",
+    "allOf": [
+        {
+            "$ref": "raw.json"
+        }
+    ]
+}
```

Comment:
Is there no difference between meta_property and regular property at all? @spectre007?